src/app/model/schema.ts

```diff
@@ -159,11 +159,13 @@ export interface FoodBoxDetails {
 export interface MissionInterface {
   id: string;
   type: MissionType;
-
+  createdDate: string; // TODO should be a date?
```

Comment:
been a bit since I talked with Tuan about it, but I think we should store dates as a unix timestamp integer. `(new Date()).getTime()`

Storing date objects always turns into a nightmare.